code/script.js

```diff
@@ -30,18 +33,138 @@ const showMessage = (message, sender) => {
   chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
+// The Bot starts here
+//The greeting function sends a message as the bot asking for your name.
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage(`Hello and welcome to Pretty Pets! <br> <br>What's the name of your furry friend?`, 'bot')
+}
+
+// The user enters dog name in the handleInput function
+// There's a condition-check to make sure that the user enters a valid name
+const handleInput = (event) => {
+  event.preventDefault()
+  // Store the value in a variable so we can access it after we clear it from the input
+  const userInput = textInput.value
+  dogName = userInput;
+  showMessage(userInput, 'user')
+  if (textInput.value === '') {
+    showMessage(`Please enter a valid name`, 'bot')
+    handleInput()
+  } 
+  textInput.value = ''
+
+
+  // After 1 second, show the next question by invoking the next function passing the name into it to have access to the user's name if we want to use it in the next question from the bot.
+  setTimeout(() => nameOfPet(userInput), 1000)
+}
+
+// Questions from the Bot about the user selection and creating two buttons (Book appointment and View treamtment)
+const nameOfPet = () => {
+  showMessage(`Would you like to book an appointment or look at our treatments?`, 'bot')
+  
+  inputWrapper.innerHTML = `
+  <button id="appointmentBtn">Book Appointment</button>
+  <button id="treatmentBtn">View Treatments</button>
+  `
+  document
+  .getElementById('treatmentBtn')
+  .addEventListener('click', () => viewTreatments())
+
+  document
+  .getElementById('appointmentBtn')
+  .addEventListener('click', () => bookTreatment())
+}
+
+// User choose View treatment and two new buttons gets displayed (Big Lebowski and Michelle Pfieffer) 
```

Comment:
Great that you added comments in the code. Makes it easy to follow. :) /Darya and Rebecca